### PR TITLE
Add homepage 

### PR DESCRIPTION
**Pull request by abredi**
- Add home page to the project
- only used html5 and css3
- used `sylelint` for linting
- used `html-validator-cli` for schematic html validations

### CODE REVIEW

**Comment by abredi** [comment]
@lucasmazo32, I already fixed that. Thanks for reminding me.

**Comment by santiago-rodrig** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,242 @@
+body,
+ul,
+nav {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  height: 100vh;
```
Comment (santiago-rodrig) [inline assets/css/main.css:10]: Maybe you want to set the **minimum height instead**.
```suggestion
  min-height: 100vh;
```

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+a {
+  text-decoration: none;
+}
+
+.billboard {
+  background-image: url(../img/your_verse_hero.jpg);
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-size: 1440px 1315px;
+  width: 100%;
```
Comment (santiago-rodrig) [inline assets/css/main.css:31]: Unnecessary
```suggestion
```

File: assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+  height: 100%;
```
Comment (santiago-rodrig) [inline assets/css/main.css:32]: Only works if parent element has a fixed height.
```suggestion
```

**Comment by abredi** [comment]
@santiago-rodrig Thanks for your review and suggestions. I fixed the responsiveness issues.

**Pull request merged**